Add allyson smith

allyson-s-code opened the pull request.

allyson-s-code committed:
Add allyson smith to contributors list

allyson-s-code committed:
Add allyson smith to contributors list

dA505819 reviewed (changes requested):
Add only your name and link to your github profile

allyson-s-code commented:
Hi,
I'm sorry, I don't think I understand.  I thought that was all I changed with the first commit but there was a merge conflict so I went back in and saw a space above my name and I thought maybe that was the conflict so I deleted it for the second commit.  At this point should I just redo it completely- fork and clone again?  
Thanks for your help!
